scripts/recording_uploader/__init__.py

```diff
@@ -0,0 +1 @@
+"""Init file for the recording_uploader package."""
```

Comment:
If this is a package, should we move it outside of `scripts`?

Reply:
It should be named as a module instead 😅